data/api/client-server/relations.yaml

```
@@ -141,7 +389,8 @@ paths:
                 title: "ChildEventsChunk"
                 type: array
                 description: |-
```

still missing its `required` flag

```suggestion
                title: "ChildEventsChunk"
                type: array
                required: true
                description: |-
```

---

right, sorry.